### PR TITLE
オーディオコーデック LYRA のサポートを削除する

### PR DESCRIPTION
**Pull request by tnamao**
## 変更履歴

- [CHANGE] オーディオコーデック `LYRA` のサポートを削除する
  - @tnamao

### CODE REVIEW

**Comment by sile** [comment]
copilot が生成した要約 

---

This pull request primarily involves the removal of support for the `LYRA` audio codec across various files. This includes changes in the `CHANGES.md` file to document this removal, modifications in the `examples/check.html` file to eliminate references to `LYRA`, and updates in the `packages/sdk/src/base.ts` and `packages/sdk/src/sora.ts` files to remove `LYRA` related code. Additionally, the `@shiguredo/lyra-wasm` package has been removed from `package.json`, and the `lyra_worker.ts` file has been deleted.

Changes are as follows:

Removal of `LYRA` support:

* [`CHANGES.md`](diffhunk://#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R14-R15): Removed support for the `LYRA` audio codec.
* [`examples/check.html`](diffhunk://#diff-b7381b614befe914569b5711418f6e1bb3f455fa1d0d7feb8abae21318b805d1L60-R60): Removed sections related to `LYRA` and replaced them with `OPUS`. [[1]](diffhunk://#diff-b7381b614befe914569b5711418f6e1bb3f455fa1d0d7feb8abae21318b805d1L60-R60) [[2]](diffhunk://#diff-b7381b614befe914569b5711418f6e1bb3f455fa1d0d7feb8abae21318b805d1L89-R81) [[3]](diffhunk://#diff-b7381b614befe914569b5711418f6e1bb3f455fa1d0d7feb8abae21318b805d1L124-R137) [[4]](diffhunk://#diff-b7381b614befe914569b5711418f6e1bb3f455fa1d0d7feb8abae21318b805d1L172-L175) [[5]](diffhunk://#diff-b7381b614befe914569b5711418f6e1bb3f455fa1d0d7feb8abae21318b805d1L205-L210)
* `packages/sdk/src/base.ts` and `packages/sdk/src/sora.ts`: Removed code related to `LYRA` and updated methods to reflect this removal. [[1]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L5-L17) [[2]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L200-L212) [[3]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L291-L293) [[4]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1311-R1286) [[5]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1463-L1483) [[6]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1493-L1499) [[7]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1509-L1570) [[8]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1584-L1630) [[9]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1311-R1286) [[10]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1463-L1483) [[11]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1493-L1499) [[12]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1509-L1570) [[13]](diffhunk://#diff-86ad7bee77842a53889b523c02bd3e7f9f8ae37a165e88d106b85e17f98628f0L1584-L1630) [[14]](diffhunk://#diff-5018df812367884fe01f345e2ca9e7ccd6f0af74ccd14e3429fbe29b08698f68L5)

Package and file deletions:

* [`packages/sdk/package.json`](diffhunk://#diff-77164cc5c071d1488e1cf72c0f91fdd99dd8976edb5004c83198bbd517039572L15): Removed the `@shiguredo/lyra-wasm` package.
* [`packages/sdk/rollup.config.mjs`](diffhunk://#diff-f942317f81a1ca01bbf706c9db873bec2f03eac2d992e0bb241ab4606214fd85L1-R6): Removed references to `lyra_worker.ts` and updated the rollup configuration. [[1]](diffhunk://#diff-f942317f81a1ca01bbf706c9db873bec2f03eac2d992e0bb241ab4606214fd85L1-R6) [[2]](diffhunk://#diff-f942317f81a1ca01bbf706c9db873bec2f03eac2d992e0bb241ab4606214fd85L16-L45) [[3]](diffhunk://#diff-f942317f81a1ca01bbf706c9db873bec2f03eac2d992e0bb241ab4606214fd85L54-R64)
* [`packages/sdk/src/lyra_worker.ts`](diffhunk://#diff-c6131af06141a8919cab99e2ecb6bc1a6929b35ac57dfc41f832bd73792d8639L1-L68): Deleted this file as it was related to `LYRA`.

File: packages/sdk/src/base.ts [hunk 1]
```diff
@@ -1490,11 +1447,8 @@ export default class ConnectionBase {
    * @returns 処理後の SDP
    */
   private processAnswerSdpForLocal(sdp: string): string {
-    if (this.lyra === undefined) {
-      return sdp
-    }
-
-    return this.lyra.processAnswerSdpForLocal(sdp)
+    // TODO(tnamao): 削除しても良さそうな関数
```
Comment (sile) [inline packages/sdk/src/base.ts:1450]: 確かにもう削除してしまってもいいと思います

File: packages/sdk/src/base.ts [hunk 2]
```diff
@@ -1581,51 +1506,24 @@ export default class ConnectionBase {
...
-        {
-          name: 'receiverTransform',
-          lyraDecoder,
-        },
-        [lyraDecoder.port],
-      )
+    // @ts-ignore
+    const receiverStreams = receiver.createEncodedStreams() as TransformStream
+    const writable = receiverStreams.writable
+    if (this.e2ee) {
```
Comment (sile) [inline packages/sdk/src/base.ts:1523]: ここの分岐も同様に不要そう

**Review by sile — APPROVED** [review]
いくつか軽微なコメントはしましたが LGTM です

**Comment by tnamao** [comment]
@sile 指摘された部分を修正しました。軽くみていただけると助かります

**Comment by sile** [comment]
対応ありがとうございます。良さそうです 👍

**Comment by tnamao** [comment]
マージします

**Pull request merged**